docs/webhook_events.md

```diff
@@ -182,6 +182,14 @@ Each event will be submitted via HTTP POST to the user provided URL.
                     "title": "Check Retry Count",
                     "type": "integer"
                 },
+                "check_fuzzer_help": {
+                    "title": "Check Fuzzer Help",
```

Comment:
We could just call this `check_fuzzer`, and potentially do other checks in the future. The fact that we invoke `-help=1` specifically seems like an implementation detail.

Reply:
The explicit setting is intentional.  This enables working towards libfuzzer-like binaries that don't use `-help=1` but have many other similar properties.